### PR TITLE
Rebuild python docker image

### PR DESCRIPTION
**Pull request by TeBoring**

### CODE REVIEW

File: tests.sh [hunk 1]
```diff
@@ -234,6 +282,66 @@ build_python_cpp() {
   cd ..
 }
 
+build_python27_cpp() {
+  internal_build_cpp
+  export LD_LIBRARY_PATH=../src/.libs # for Linux
+  export DYLD_LIBRARY_PATH=../src/.libs # for OS X
+  cd python
+    envlist=py27-cpp
```
Comment (haon4) [inline tests.sh]: why is this indented? Same comment below

File: tests.sh [hunk 2]
```diff
@@ -220,6 +220,54 @@ build_python() {
   cd ..
 }
 
+build_python27() {
```
Comment (haon4) [inline tests.sh]: most of these functions are the same, can we call a helper function and just pass in the envlist as an argument?

Comment (TeBoring) [reply tests.sh]: done

File: tests.sh (hunk 1 above)
Comment (TeBoring) [reply tests.sh]: Removed indent.

**Pull request merged**